firestore.rules

```diff
@@ -51,7 +56,7 @@ service cloud.firestore {
     // check if it's the user's own data
     function isUsersData(userId) {
       return currentUser().uid == userId
```

Comment:
this doesn't jive with currentUser either. This is essentially `request.auth.uid.uid`